<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>weather</class>
 <widget class="QDialog" name="weather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天气查询</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>258</width>
     <height>233</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>温州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>宁波</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
